diff --git a/picade.dts b/picade.dts
index 81792b4..e3866fa 100644
--- a/picade.dts
+++ b/picade.dts
@@ -69,37 +69,37 @@
 
 				button1: button1 {
 					label = "Button 1";
-					linux,code = <29>;
+					linux,code = <31>;
 					gpios = <&gpio 5 1>;
 				};
 
 				button2: button2 {
 					label = "Button 2";
-					linux,code = <56>;
+					linux,code = <32>;
 					gpios = <&gpio 11 1>;
 				};
 
 				button3: button3 {
 					label = "Button 3";
-					linux,code = <57>;
+					linux,code = <33>;
 					gpios = <&gpio 8 1>;
 				};
 
 				button4: button4 {
 					label = "Button 4";
-					linux,code = <42>;
+					linux,code = <36>;
 					gpios = <&gpio 25 1>;
 				};
 
 				button5: button5 {
 					label = "Button 5";
-					linux,code = <44>;
+					linux,code = <37>;
 					gpios = <&gpio 9 1>;
 				};
 
 				button6: button6 {
 					label = "Button 6";
-					linux,code = <45>;
+					linux,code = <38>;
 					gpios = <&gpio 10 1>;
 				};
 
